spk/synokernel-linuxtv/src/synokernel-linuxtv.cfg

```diff
@@ -1,6 +1,6 @@
 # see also: /lib/udev/devicetable/usb.DVB.dep.table
-default="regmap-i2c rc-core mc videodev tveeprom videobuf2-common videobuf2-v4l2 videobuf2-memops videobuf2-vmalloc dvb-core dvb-usb dvb_usb_v2 uvcvideo"
-HAUPPAUGE_WINTV_DUALHD="rc-hauppauge si2157 lgdt3306a em28xx em28xx-dvb em28xx-rc"
-PCTV461E="rc-pinnacle-pctv-hd ts2020 a8293 m88ds3103 em28xx em28xx-dvb em28xx-rc"
-MYGICA_T230="rc-dvbsky si2157 si2168 m88ds3103 dvb-usb-dvbsky"
-SMS_SIANO_MDTV="smsmdtv smsdvb smsusb"
+default:"regmap-i2c rc-core mc videodev tveeprom videobuf2-common videobuf2-v4l2 videobuf2-memops videobuf2-vmalloc dvb-core dvb-usb dvb_usb_v2 uvcvideo dvb-usb-rtl28xxu"
+#HAUPPAUGE_WINTV_DUALHD:"rc-hauppauge si2157 lgdt3306a em28xx em28xx-dvb em28xx-rc"
+#PCTV461E:"rc-pinnacle-pctv-hd ts2020 a8293 m88ds3103 em28xx em28xx-dvb em28xx-rc"
+#MYGICA_T230:"rc-dvbsky si2157 si2168 m88ds3103 dvb-usb-dvbsky"
+#SMS_SIANO_MDTV:"smsmdtv smsdvb smsusb"
```

Comment:
The `default` is always being included when loading any other lines.  In your case here you should rather create a new line specifically for this adapter module loading.

Looking into the kernel the `Makefile` entry name is `CONFIG_DVB_USB_RTL28XXU` to which in this case I'd use:
```
RTL28XXU="rtl28xxu"
```

So when calling the loading script with `RTL28XXU` as config option to use, it first loads all modules from the `default` line then will load this specific mode as well.

Reply:
Ok makes sense